pkg/logger/handler.go

```diff
@@ -91,7 +91,7 @@ func (eh *LoggerHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 			Url:              eh.logUrl,
 			Bytes:            &body,
 			ContentType:      contentType,
-			ReqType:          InferenceRequest,
+			ReqType:          CEInferenceRequest,
```

Comment:
this changes from `Request` to CEInferenceRequest, does it break anything?

Reply:
nvm I see it is setting to CEInferenceRequest later on, so we can remove InferenceRequest type